guardian/fence
# Prepare package for publishing to NPM

## What does this change?

This PR includes minimal changes to get the package ready for publishing on [npm](https://www.npmjs.com/package/@guardian/fence).

Changes include:
1. Added `@guardian` namespace to package name.
2. Moved publish-ignored files from `bower.json/ignore` to `.npmignore`
3. Updated install instructions in `README.md` to refer to npm instead of bower,
4. Removed `bower.json`



## How to test



Update the package from npm: `npm i @guardian/fence` and ensure it correctly installs.
Additional step: Update an existing repository that uses fence to fetch the dependency from npm, instead of directly from github/bower.

## How can we measure success?

The  dependencies provided by https://github.com/guardian/mobile-apps-article-templates/blob/master/package.json in https://github.com/guardian/android-news-app should start being available again in the [TeamCity builds](https://teamcity.gutools.co.uk/buildConfiguration/Apps_Android_AndroidNewsAppPullRequests?mode=builds).


## Have we considered potential risks?



## Images



## Accessibility
